process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
 | underscoreMetadata | Boolean | *Optional.* If set to `true`, the input `targetFileMetadata` can have its namespace prefixes written with `_` instead of `:`, for example `cm_title` instead of `cm:title`. This allows the JSON to be used in an expression, for example `${metadata.cm_title}`, whereas `${metadata.cm:title}` is not valid. |
 | targetFileType | Content-Type | *Optional.* The type to set for the signed file, for example `fin:invoice`. |
 | targetFile | File | *Requires one.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type file that should be updated with the signed version of the document. |
 | targetFolder | Folder | *Requires one.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type folder to store the signed document in. |
 | targetFolderId | String | *Requires one.* The nodeId of the folder to store the signed document in. For example `775a8f2d-8123-49a7-ae1f-f3f49d4eae20`. |
 | targetFolderPath | String | *Requires one.* The location path or relative path of the folder to store the signed document in. For example, a location path: `/app:company_home/app:user_homes/cm:hruser` and a relative path: `/User Homes/hruser`. |
 | timeout | Integer | *Optional.* The timeout period to wait for the document to be signed in milliseconds, for example `910000`. |
-| signers | JSON | *Optional.* The list of signers who sign the document. |
+| signers | JSON | *Optional.* The list of signers (including email, name, sign here page, sign here tab label, position X and position Y) when the document is going to be signed by more than one signer.. |
 | allowMarkup | Boolean | *Optional.* Allow recipients to make changes to your documents by covering up existing text and replacing it with new text (i.e. markup). Recipients can decide to use a special markup text field which they can place anywhere on the document. It can be scaled and optionally filled in. All changes must be reviewed and approved by all signers. |
+| file | File | *Required* File to be signed. If multiple files are received, only the first one will be processed. |
 
-> **Note**: `underscoreMetadata` can be set to `true` and the `targetFileMetadata` input can still use `:` with the connector successfully executing the action. If `underscoreMetadata` is set to `false` and `targetFileMetadata` uses `_` then the connector will fail to execute the action.
+The output parameters from `SEND_FOR_SIGNATURE` are:
+
+| Parameter | Type | Description |
+|--- | --- | --- |
+| envelopeId | String | Envelope ID of the document. |
+| status | String | Status of the envelope. |
+| URI | String | URI related to the envelope. |
+| docusignOutput | JSON | DocuSign output after sending the document for signature. |
 
-An example of the `targetFileMetadata` that can be stored with the document is:
+The input parameters for `DOWNLOAD_DOCUMENT` are:
```

Comment:
Same as above. What do you think if we include a title for each action like in the rest of the connectors? As we had it before?

Reply:
Done.